bakery/env/local.json

```diff
@@ -5,5 +5,9 @@
   "S3_ACCESS_KEY_ID": "",
   "S3_SECRET_ACCESS_KEY": "",
   "S3_DIST_BUCKET": "",
-  "VERSIONED_FILE": ""
+  "S3_VERSIONED_BUCKET": "",
+  "VERSIONED_FILE": "",
+  "FEED_FILE_URL": "",
+  "PIPELINE_TICK_INTERVAL": "",
+  "MAX_BOOKS_PER_TICK": ""
```

Comment:
We decided that if we needed more responsiveness, we should add a `pending` state so that we could queue books at some shorter time interval, but that if we didn't the equivalent was queuing all books at a large time interval. If this is the case, should we not need `MAX_BOOKS_PER_TICK` (or we should set it to a value greater than the number of books we expect on the list) until such time arrives that we need a `pending` state and higher responsiveness?

Reply:
I just piped it through for now since if nothing else it is useful for development / testing already to be able to set this value. I agree that initially the production value for this setting should be something high, but left it blank for now as I wasn't sure of a good value just yet. I can set something in `prod.json`, though, if there's a number you think makes sense to put for now.

Reply:
As long as we know to set it high for prod, I'm :ok_hand: